guides/release/configuring-ember/optional-features.md

```diff
@@ -196,10 +196,10 @@ can be set to run asynchronously using the `sync: false` option.
 import { observer } from '@ember/object';
 
 Person.extend({
-  partOfNameChanged: observer({
-    dependentKeys: ['firstName', 'lastName'],
+  aspectRatioChanged: observer({
```

Comment:
I think it's a bit odd that the observer name is `aspectRatioChanged`, yet the dependent keys are `width` and `height` (and not `aspectRatio`).

What do you think of naming the observer `onImageSizeChange` or `onScreenSizeChange`?

Reply:
Addressed in 39a216a8